### DISCUSSION TITLE
7.3.0 cy.intercept - how regex url matcher works?

### DISCUSSION THREAD

**Discussion by debasisj** [cypress-io/cypress#16492] [Questions and Help]
Hi Guys, 
I just upgraded Cypress to `7.3.0` and my `intercept` asserts had to go thru quite bit changes.
This is the POST call API does - `https://api.env.com.au/customers/randomId/verification`
In cypress `6.x` this used to work `  cy.intercept('POST', '/verification').as('verification')`, but it doesn't work in latest 
I have gone thru the latest document and changed to this 
```js
   cy.intercept({method: 'POST',url: /^https:\/\/api\.env\.com\.au\/customers\/.*\/verification/}).as('verification')

   cy.get('.lpResendPhoneVerifCode').last().click()

  cy.wait('@verification').its('response.statusCode').should('eq', 200)
```
This works but, can it be simpler ..?

**Comment by jennifer-shehane** [comment]
This regex may not be necessary. 

What is the exact issue when you write the code like below? Do you get an error? Do you have any other intercepts defined before that may be also matching?

```js
cy.intercept('POST', '/verification').as('verification')
```
